rtp/README.md

```diff
@@ -32,11 +32,8 @@ You should be able to see an SDL player showing an example video.
 The stream can also be sent with GStreamer:
```

Comment:
Hmm I believe we should prepare more complex readme - 
i.e. we should explicitly say which part of the system does the gstream substitue, as well as I would provide some links to the dependencies that need to be installed (even to fixed versions of these dependencies). We can also think about adding a brief description of what is going on within `receive.exs` and `send.exs`.